<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CharacterCreatorForm</class>
 <widget class="QWidget" name="CharacterCreatorForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>389</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="1" colspan="3">
    <widget class="QGroupBox" name="groupBoxCreatePlayer">
     <property name="title">
      <string>Create a character</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Name:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="1" colspan="2">
         <widget class="QLineEdit" name="lineEditName">
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>Let empty for a name random</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Server:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="comboBoxServer">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Breed:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="comboBoxBreed">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QCheckBox" name="checkBoxRandomBreed">
          <property name="text">
           <string>Random</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Sex:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QComboBox" name="comboBoxGender">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <item>
           <property name="text">
            <string>Mal</string>
           </property>
           <property name="icon">
            <iconset resource="../../../../assets/resources.qrc">
             <normaloff>:/icons/male.ico</normaloff>:/icons/male.ico</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Female</string>
           </property>
           <property name="icon">
            <iconset resource="../../../../assets/resources.qrc">
             <normaloff>:/icons/female.ico</normaloff>:/icons/female.ico</iconset>
           </property>
          </item>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QCheckBox" name="checkBoxRandomGender">
          <property name="text">
           <string>Random</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Head:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QComboBox" name="comboBoxHead">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="iconSize">
           <size>
            <width>48</width>
            <height>48</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QCheckBox" name="checkBoxRandomHead">
          <property name="text">
           <string>Random</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="checkBoxRandomColor">
        <property name="text">
         <string>Couleur al√©atoire</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QPushButton" name="pushButtonSkin">
          <property name="text">
           <string>Skin</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="pushButtonHair">
          <property name="text">
           <string>Hair</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="pushButtonClothing_1">
          <property name="text">
           <string>Clothing 1</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="pushButtonClothing_2">
          <property name="text">
           <string>Clothing 2</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="pushButtonClothing_3">
          <property name="text">
           <string>Clothing 3</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="pushButtonResetColor">
          <property name="text">
           <string>Reset the color</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="1">
         <widget class="QCheckBox" name="checkBoxSkipDidactiel">
          <property name="text">
           <string>Skip didactiel</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>43</width>
            <height>18</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="0">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>87</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" rowspan="3">
    <widget class="QListWidget" name="listWidgetAccounts">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>86</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="pushButtonSave">
     <property name="text">
      <string>Save</string>
     </property>
     <property name="icon">
      <iconset resource="../../../../assets/resources.qrc">
       <normaloff>:/icons/diskette_16px.ico</normaloff>:/icons/diskette_16px.ico</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>listWidgetAccounts</tabstop>
  <tabstop>groupBoxCreatePlayer</tabstop>
  <tabstop>lineEditName</tabstop>
  <tabstop>comboBoxServer</tabstop>
  <tabstop>comboBoxBreed</tabstop>
  <tabstop>comboBoxGender</tabstop>
  <tabstop>comboBoxHead</tabstop>
  <tabstop>checkBoxRandomBreed</tabstop>
  <tabstop>checkBoxRandomGender</tabstop>
  <tabstop>checkBoxRandomHead</tabstop>
  <tabstop>checkBoxRandomColor</tabstop>
  <tabstop>pushButtonSkin</tabstop>
  <tabstop>pushButtonHair</tabstop>
  <tabstop>pushButtonClothing_1</tabstop>
  <tabstop>pushButtonClothing_2</tabstop>
  <tabstop>pushButtonClothing_3</tabstop>
  <tabstop>pushButtonResetColor</tabstop>
  <tabstop>checkBoxSkipDidactiel</tabstop>
  <tabstop>pushButtonSave</tabstop>
 </tabstops>
 <resources>
  <include location="../../../../assets/resources.qrc"/>
 </resources>
 <connections/>
</ui>
